core/src/test/scala/com/yahoo/maha/core/RequestModelTest.scala

```diff
@@ -5930,5 +5930,29 @@ class RequestModelTest extends FunSuite with Matchers {
     val res = RequestModel.from(request, registry)
     assert(res.isSuccess, s"should not fail on having filter on Advertiser ID")
   }
+
+  test("Should fail without any part of required filtering list") {
+    val jsonString = s"""{
```

Comment:
Can we add a test to make sure only one of the filter columns is required, not all.